react/react-state-2_favourite-holiday/src/App.js

```diff
@@ -21,19 +25,25 @@ export default function App() {
           type="text"
           name="holiday"
           placeholder="e.g. Christmas"
+          onChange={(holiday) => setHoliday(holiday.target.value)}
```

Comment:
you made the setter do its job directly in the input tag, it asked for:
The handleSubmit function sets both states with its corresponding values. Do you remember how to access the values of the target fields?